fairlearn/reductions/exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -65,7 +65,20 @@ def __init__(self, estimator, constraints, eps=0.01, T=50, nu=None, eta_mul=2.0)
 
     def fit(self, X, y, **kwargs):
         # TODO: validate input data; unify between grid search and expgrad?
-        self._expgrad_result = exponentiated_gradient(X, kwargs[_KW_SENSITIVE_FEATURES], y,
+
+        if type(X) in [np.ndarray, list]:
```

Comment:
Not necessary right now, but this (and the next if block) should probably be in a `utils` library and used in other methods

Reply:
Absolutely! Just needed a couple of things here to get it running. I want to tackle this validation separately (or you, whoever gets to it first)